src/chainparams.cpp

```diff
@@ -271,6 +275,8 @@ class CRegTestParams : public CChainParams {
         consensus.BIP34Hash = uint256();
         consensus.BIP65Height = 1351; // BIP65 activated on regtest (Used in rpc activation tests)
         consensus.BIP66Height = 1251; // BIP66 activated on regtest (Used in rpc activation tests)
+        consensus.BTGHeight = 100000000; // Not activated yet.
```

Comment:
checks out

Reply:
This is the height for regression test mode. As we didn't write a regression test for the hard fork yet, I left it a very large number. Thought?